tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Deduplicaation/DeduplicationTests.cs

```diff
@@ -0,0 +1,54 @@
+// <copyright file="DeduplicationTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.Iast;
```

Comment:
not sure where to add the nit comment but the folder's name  is ''Deduplicaation" with 2 a's.

Reply:
Good catch! I have corrected the name.